vignettes/philosophy.Rmd

```diff
@@ -0,0 +1,163 @@
+---
+title: "Philosophy of the checklist package"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Philosophy of the checklist package}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+references:
+- type: article-journal
+  id: Baath2012
+  author:
+  - family: Bååth
+    given: Rasmus
+  issued:
+    date-parts:
+    - - 2012
+      - 12
+  title: The State of Naming Conventions in R
+  container-title: The R Journal
+  volume: 4
+  number: 2
+  page: 74--75
+  doi: 10.32614/RJ-2012-018
+  URL: https://doi.org/10.32614/RJ-2012-018
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+Everybody develops their own coding habits and style.
```

Comment:
Everybody is singular, so develops was written correct, but their should be replaced by a singular renown
```suggestion
Everybody develops his/her own coding habits and style.
```